doc/getting_started.dox

```diff
@@ -110,9 +131,99 @@
+ * @code
+ * $ls -l
+ * total 31124
+ * drwxr-sr-x 7 root staff     4096 Sep 10 15:19 db
+ * drwxr-sr-x 2 root staff     4096 Sep 10 15:19 help
+ * drwxr-sr-x 2 root staff     4096 Sep 10 15:19 olddb
+ * drwxr-sr-x 3 root staff     4096 Sep 10 15:19 plugins
+ * lrwxrwxrwx 1 root staff       24 Sep 18 21:32 STM32Cube_FW_F3 -> STM32Cube_FW_F3_V1.10.0/
+ * drwxr-sr-x 8 root staff     4096 Sep 10 15:41 STM32Cube_FW_F3_V1.10.0
+ * -rwxr-xr-x 1 root staff 31846117 Sep 10 15:19 STM32CubeMX
+ * drwxr-sr-x 2 root staff     4096 Sep 10 15:19 Uninstaller
+ * $ 
+ * @endcode
+ *
+ * Now, you should be ready to compile!
+ * 
+ *
+ * @subsubsection somemake Some pointers about the make command
+ * Make is powerful and when setup, makes it easy to build software from source. You have to keep in mind that it is running what a Windows user might think of as a batch file; properly here in the Unix world, a shell script, so it won’t know everything that has gone on, so a good process is helpful for best results.
+ *
+ * @code
+ * $make clean
+ * @endcode
+ * Deletes the files that a make creates; use this after you tweak the source, but before you recompile with a plain make. See the next step.
+ *
+ * @code
+ * $make
+ * @endcode
+ * Will just commence the software build, according the makefile. You should not have to mess with the makefile so if the output from make is a bunch of errors, check your paths and libraries. The openmrn build system is good about giving hints on what’s missing.
+ *
+ * @subsection build Building the software
+ * Compile the `blink_raw` application for the
+ * `freertos-armv6m-st-stm32f091rc-nucleo` target like this:
+ * @code
+ * $ cd openmrn/applications/blink_raw/targets/freertos-armv6m-st-stm32f091rc-nucleo
+ * $ make
+ * @endcode
+ *
+ * If you have troubles getting the source to compile, don't set eny 
+ * environment variables but look closely to the location of the files and  
+ * directories because most likely a mistake is made there.
+ *
+ * The output of the make process will leave you with a *.elf file. To make it a binary that you can drop onto the Nucleo, run:
+ * @code
+ * $objcopy -O binary blink_raw.elf blink_raw.bin
+ * @endcode
+ *
+ * @subsubsection devkit DevKit IO and Nucleo
+ * If you are using the DevKit IO board and the Nucleo, you will want to compile the application at openmrn/applications/nucleo_io and the appropriate target in the target/ directory you will find under it.
+ * @code
+ * openmrn/applications/nucleo_io/targets$ ls -l
+total 12
+ * drwxr-xr-x 2 rick rick 4096 Sep 17 20:35 freertos.armv6m.st-stm32f091rc-nucleo-dev-board
+ * drwxr-xr-x 3 rick rick 4096 Sep 18 21:52 freertos.armv7m.st-stm32f303re-nucleo-dev-board
+ * -rw-r--r-- 1 rick rick  172 Sep  7 20:31 Makefile
+ * $cd freertos.armv7m.st-stm32f303re-nucleo-dev-board/
+ * $make
+ * @endcode
+ * As before, you will get a .elf file, nucleo_io.elf, so the objcopy command is called for here.
```

Comment:
Rick, this looks great, I think you got everything we ran into.  Glad you remembered that note for the GCC build tools.  That one had me scratching my head for a few hours.  I have some stuff on setting up OpenLCB I will add after this gets merged.